### ISSUE TITLE
Allow setting variables from args

### ISSUE THREAD

**Issue by johanhelsing** [YarnSpinnerTool/YarnSpinner-Console#15]
The `--start-node` option is pretty useful for testing parts of a conversation. It would be even more useful if it was also possible to set variables.

I.e. something like

```
ysc run main.yarn --start-node some_deep_node --set-variable met_sally=true --set-variable money=123
```

**Comment by McJones** [comment]
This is a cool idea, I am just trying to work out the best way to facilitate this.
Is it better to inject variables live like this or maybe some sort of config file? Or does that lose the "oh wait actually met_sally should be false" feeling?

I suppose at least initially the easiest way to get this would be to create a new node with your variables set and enter that way. Does feel like a bit of an extra step though if you have to edit the yarn.

**Comment by johanhelsing** [comment]
Yeah, i currently have a "debug start menu" node, that just lets me enable stuff, then loops back to itself, then starts what i really intended to test.

It's a bit tedious, but kinda works. Args or perhaps special commands while running would be optimal for my use case.
